cpp/modmesh/CMakeLists.txt

```diff
@@ -8,13 +8,17 @@ if (BUILD_QT)
     set(CMAKE_AUTORCC ON)
     set(CMAKE_AUTOUIC ON)
 
-    find_package(Qt6 REQUIRED COMPONENTS Core)
-    find_package(Qt6 REQUIRED COMPONENTS Widgets)
-    find_package(Qt6 REQUIRED COMPONENTS Gui)
-    find_package(Qt6 REQUIRED COMPONENTS 3DCore)
-    find_package(Qt6 REQUIRED COMPONENTS 3DRender)
-    find_package(Qt6 REQUIRED COMPONENTS 3DInput)
-    find_package(Qt6 REQUIRED COMPONENTS 3DExtras)
+    find_package(Qt6 QUIET)
+    if(NOT Qt6_FOUND)
+        message(FATAL_ERROR  "Qt6 Not Found. Try `cmake -DCMAKE_PREFIX_PATH=/path/to/Qt/6.3.2/gcc_64/lib/cmake`?")
```

Comment:
In an error message it's probably more elegant to not specifying minor versions, like:

```
/path/to/Qt/6.x.y/<platform>
```

And I think `lib/cmake` is redundant because `CMAKE_PREFIX_PATH=/path/to/qt/6.2.3/macos` works for me.  If it is not the case in Linux or Windows, could you add a couple of more words to make the message complete?

Reply:
in Windows and Linux, the path is different. I can just list three paths for all platforms.

Reply:
Listing 3 paths sounds mouthy.  How about just indicate that "mac does not need lib/cmake"?

Reply:
another way is to use if condition to print messages according to the platform, I think it would be clearer for user.

Reply:
That sounds about right.